ruby_scripts/common/campaign.rb

```diff
@@ -34,7 +34,10 @@ def qualifies?(cart)
       end rescue nil
       if is_selector
         raise "Missing line item match type" if @li_match_type.nil?
-        cart.line_items.send(@li_match_type) { |item| qualifier.match?(item) }
+        cart.line_items.send(@li_match_type) do |item|
+          next false if item.nil?
+          qualifiers.match?(item)
```

Comment:
This should be `qualifier` instead.
```suggestion
          qualifier.match?(item)
```

Reply:
My bad, surprised nvim did not pick up on that.